tsl/test/sql/bgw_custom.sql

```diff
@@ -51,7 +51,7 @@ SELECT add_job('custom_proc2','1h', config:= '{"type":"procedure"}'::jsonb);
 SELECT add_job('custom_func', '1h', config:='{"type":"function"}'::jsonb);
 SELECT add_job('custom_func_definer', '1h', config:='{"type":"function"}'::jsonb);
 
-SELECT * FROM timescaledb_information.jobs WHERE job_id != 1 ORDER BY 1;
+SELECT * FROM timescaledb_information.jobs WHERE job_id NOT IN (1,2) ORDER BY 1;
```

Comment:
Why 1 and 2? Please add a comment what these jobs are and why you need to exclude them from the result.

Reply:
Looks good. Resolving comment.